test/webhook_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
It's not always 100% necessary but I see lots of people add `set -e` and `set -o pipefail` so the script will exit unclean with any unexpected error and also will catch `piped` errors (in a scenario where a piped shell command can mask an exit code. [More info here](https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail)

Reply:
I don't think we want to `set -e` here, since we're expecting some commands to fail. `set -o pipefail` seems safe though.

You could also `set -e` for the setup, then `set +e` before you start running the tests.

Reply:
The `set -e` is actually a safer mechanism for testing to catch all unexpected errors. If will converted his tests to be `if kubectl...` instead of `kubectl ... ; if [ $? -ne 0 ]; then...` - it would actually work fine with `set -e`

From the docs: 
>The shell does not exit if the command that fails is part of the command list immediately following a while or until keyword, part of the test in an if statement, part of any command executed in a && or || list except the command following the final && or ||, any command in a pipeline but the last, or if the command’s return status is being inverted with !.
[docs](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html)